tests/js/spec/views/settings/organizationIntegrations/index.spec.jsx

```diff
@@ -265,6 +265,27 @@ describe('OrganizationIntegrations', () => {
       });
     });
 
+    describe('internal apps are separate', () => {
+      it('renders internal sentry app', () => {
+        const internalApp = {...sentryApp, status: 'internal'};
+        Client.addMockResponse({
+          url: `/organizations/${org.slug}/sentry-apps/`,
+          body: [internalApp],
+        });
+        Client.addMockResponse({
+          url: '/sentry-apps/',
+          body: [],
+        });
+        wrapper = mount(
+          <OrganizationIntegrations organization={org} params={params} />,
+          routerContext
+        );
+        expect(
+          wrapper.find('Panel [data-test-id="internal-integration-row"]').exists()
+        ).toBe(true);
```

Comment:
Can we add another assertion to ensure it displays "Internal" in this row? Just in case that logic gets messed with.